packages/plugin-verify-dependencies/src/profile/0.75.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Imports the base profile configuration from version 0.74
+ */
+import {default as profile074} from './0.74';
+
+/**
+ * Default export containing dependency specifications and configurations
+ * @exports {Object} Configuration object extending profile074
+ */
+export default {
+  ...profile074,
+  /**
```

Comment:
react updates are missing in this version

Reply:
yeah we can bump to 18.3.1